Vagrantfile

```diff
@@ -102,7 +102,7 @@ Vagrant.configure('2') do |config|
     config.vm.synced_folder synced_folder.fetch('local_path'), synced_folder.fetch('destination'), options
   end
 
-  config.vm.provision provisioner do |ansible|
+  config.vm.provision "drupalvm", type: "ansible"  do |ansible|
```

Comment:
It looks like rubocop is angry about this formatting:

```
1.32s$ rubocop ./Vagrantfile ./lib/drupalvm --except LineLength,Eval,MutableConstant,BlockLength,MethodLength,ConditionalAssignment,IndentArray,AlignParameters
Inspecting 2 files
C.
Offenses:
Vagrantfile:105:23: C: Prefer single-quoted strings when you don't need string interpolation or special symbols.
  config.vm.provision "drupalvm", type: "ansible"  do |ansible|
                      ^^^^^^^^^^
Vagrantfile:105:41: C: Prefer single-quoted strings when you don't need string interpolation or special symbols.
  config.vm.provision "drupalvm", type: "ansible"  do |ansible|
                                        ^^^^^^^^^
Vagrantfile:105:50: C: Unnecessary spacing detected.
  config.vm.provision "drupalvm", type: "ansible"  do |ansible|
                                                 ^
2 files inspected, 3 offenses detected
The command "rubocop ./Vagrantfile ./lib/drupalvm --except LineLength,Eval,MutableConstant,BlockLength,MethodLength,ConditionalAssignment,IndentArray,AlignParameters" exited with 1.
```

See build: https://travis-ci.org/geerlingguy/drupal-vm/jobs/290870836#L508-L527